public/styles.css

```diff
@@ -926,6 +926,9 @@ input[type=number]::-webkit-outer-spin-button {
     position: absolute;
     z-index: 10000;
     box-shadow: 0 0 2rem 0 black;
+   
+    /* Re position dropdown to avoid layout break*/
+    top: 7%;
```

Comment:
I think with the said percentage its safer instead of using a defined number. I am not defining a height for the dropdown and i don't think there should be an instance where the said 7% would not be enough. When you defined a top, it always measure from the parent container viewport. However, in this case i only measure and assumed. Should there be more accurate number we can use perhaps ? 😉